addon/components/g-map/advanced-marker.js

```diff
@@ -0,0 +1,29 @@
+import TypicalMapComponent from './typical-map-component';
+import { toLatLng } from '../../utils/helpers';
+
+export default class AdvancedMarker extends TypicalMapComponent {
+  get name() {
+    return 'advanced-markers';
+  }
+
+  get newOptions() {
+    if (!this.args.position) {
+      this.options.position = toLatLng(this.args.lat, this.args.lng);
+    }
+    if (this.args.draggable) {
+      this.options.gmpDraggable = this.args.draggable;
+    }
```

Comment:
I try to avoid introducing new options as much as possible. As annoying as the naming is, if it's called `gmpDraggable` in Google's docs, then that's what people should use.

Reply:
You're right. 👍